libs/vm-modules/include/vm_modules/ml/model/model_estimator.hpp

```diff
@@ -239,6 +253,7 @@ class ModelEstimator
     // Model
     DataType forward_pass_cost{0.0};
     DataType backward_pass_cost{0.0};
+    DataType metrics_cost{0.0};
```

Comment:
all of these can now be string initialisations (maybe coordinate with Jiri for the nicest way to do this for t he static constexpr functions above)